backends/arm/operators/op_add.py

```diff
@@ -45,6 +45,12 @@ def define_node(
         # Handle int8 (quantized) and int32
         assert inputs[0].dtype in [ts.DType.INT8, ts.DType.INT32]
 
+        dim_order = (
```

Comment:
I guess we already assumed that the type of dim-order (and also rank) is same earlier.
By broadcasting to a larger dim-order helps with different rank. But for completeness, do you want to add an assert to make sure the dim-orders are of same type?